FunctionalTableData/TableView/FunctionalTableData.swift

```diff
@@ -113,7 +114,7 @@ public class FunctionalTableData: NSObject {
 			let reload: UITableViewRowAnimation
 			public static let `default` = Actions(insert: .fade, delete: .fade, reload: .automatic)
 			public static let legacy = Actions(insert: .top, delete: .top, reload: .automatic)
-
+			
```

Comment:
Same about these

Reply:
See comment above regarding indentation in the rest of the file